### ISSUE TITLE
Typo in air_llm setup.py

### ISSUE THREAD

**Issue by volkerjaenisch** [lyogavin/Anima#48]
Great work you are doing here!

```
   install_requires=[  # I get to this in a second
        'tqdm',
        'torch',
        'transformers',
        'accelerate',
        'safetensors',
        'optimum',
        'huggingface_hub'
        'scipy',
        #'bitsandbytes' set it to optional to support fallback when not installable
    ],

->>

        'huggingface_hub',

```
Cheers,
Volker

**Comment by lyogavin** [comment]
Thanks!

**Comment by lyogavin** [comment]
fixed in [2b5d50b](https://github.com/lyogavin/Anima/commit/2b5d50b9caa10ad3765398f65209f19a6804a52a)
